Kickstarter-iOS/Views/Controllers/ThanksViewControllerTests.swift

```diff
@@ -32,7 +32,7 @@ class ThanksViewControllerTests: TestCase {
         let project = Project.cosmicSurgery
           |> Project.lens.id .~ 3
 
-        let controller = ThanksViewController.configured(with: (project, Reward.template, nil))
+        let controller = ThanksViewController.configured(with: (project, Reward.template, nil, .newPledge))
```

Comment:
I don't think we should be hard-coding .newPledge here. There are some comments below that suggest a re-factor that should remove the need for this parameter.